src/websocket_service/mod.rs

```diff
@@ -112,6 +119,57 @@ async fn handle_msg(
     Ok(())
 }
 
+async fn resubscribe(
+    database: &Arc<Database>,
+    client: &Arc<relay_client::websocket::Client>,
+) -> Result<()> {
+    info!("Resubscribing to all topics");
+    // Get all topics from db
+    let cursor = database
+        .collection::<LookupEntry>("lookup_table")
+        .find(None, None)
+        .await?;
+
+    // Iterate over all topics and sub to them again using the _id field from each
+    // record
+    // Chunked into 500, as thats the max relay is allowing
+    cursor
+        .chunks(500)
+        .for_each(|chunk| {
+            let topics = chunk
+                .into_iter()
+                .filter_map(|x| x.ok())
+                .map(|x| Topic::new(x.topic.into()))
+                .collect::<Vec<Topic>>();
+            if let Err(e) = executor::block_on(client.batch_subscribe(topics)) {
+                warn!("Error resubscribing to topics: {}", e);
+            }
+            future::ready(())
+        })
+        .await;
+
+    let cursor = database
+        .collection::<ProjectData>("project_data")
+        .find(None, None)
+        .await?;
+
+    cursor
+        .chunks(500)
```

Comment:
You can use the const from from the `relay_rpc`:
https://github.com/WalletConnect/WalletConnectRust/blob/main/relay_rpc/src/rpc.rs#L27

Reply:
Done, good catch, thanks!